ssr/src/page/account_help.rs

```diff
@@ -0,0 +1,110 @@
+                ></path>
+            </svg>
+        </div>
+    }
+}
+
+#[component]
+pub fn AccountHelp() -> impl IntoView {
+    let (is_connected, _) = account_connected_reader();
+    let show_delete_modal = create_rw_signal(false);
+    let show_confirm_delete_modal = create_rw_signal(false);
+    let show_confirm_perma_delete_modal = create_rw_signal(false);
+
+    view! {
+        <ShadowOverlay show=show_confirm_delete_modal>
```

Comment:
move the delete modal to a separate component so the AccountHelp component is easier to read

Reply:
aye, on it